test/functional/page_objects/visualize_page.js

```diff
@@ -332,34 +331,41 @@ export function VisualizePageProvider({ getService, getPageObjects, updateBaseli
       await PageObjects.header.waitUntilLoadingHasFinished();
     }
 
+    async isChecked(selector) {
+      const checkbox = await testSubjects.find(selector);
+      return await checkbox.isSelected();
+    }
+
     async checkCheckbox(selector) {
-      const element = await testSubjects.find(selector);
-      const isSelected = await element.isSelected();
-      if(!isSelected) {
+      const isChecked = await this.isChecked(selector);
+      if (!isChecked) {
         log.debug(`checking checkbox ${selector}`);
         await testSubjects.click(selector);
       }
     }
 
     async uncheckCheckbox(selector) {
-      const element = await testSubjects.find(selector);
-      const isSelected = await element.isSelected();
-      if(isSelected) {
+      const isChecked = await this.isChecked(selector);
+      if (isChecked) {
         log.debug(`unchecking checkbox ${selector}`);
         await testSubjects.click(selector);
       }
     }
 
     async setSelectByOptionText(selectId, optionText) {
       const options = await find.allByCssSelector(`#${selectId} > option`);
-      const optionsTextPromises = options.map(async (optionElement) => {
+      const optionsTextPromises = options.map(async optionElement => {
         return await optionElement.getVisibleText();
       });
       const optionsText = await Promise.all(optionsTextPromises);
 
       const optionIndex = optionsText.indexOf(optionText);
       if (optionIndex === -1) {
-        throw new Error(`Unable to find option '${optionText}' in select ${selectId}. Available options: ${optionsText.join(',')}`);
+        throw new Error(
+          `Unable to find option '${optionText}' in select ${selectId}. Available options: ${optionsText.join(
+            ','
```

Comment:
isn't that a bit weird formatting?

Reply:
Yeah, a bit. I was just letting the prettier gods do what they saw fit.

Extracted the offending `join`'d array into a variable so that it doesn't get wrapped like this.